man/docker-build.1.md

```diff
@@ -142,6 +143,12 @@ two memory nodes.
   If the path is not absolute, the path is considered relative to the `cgroups` path of the init process.
 Cgroups are created if they do not already exist.
 
+**--ulimit**=[]
+  Ulimit options
+
+  For more information about ulimit see [Setting ulimits in a 
+container](../docs/reference/commandline/run.md#setting-ulimits-in-a-container)
```

Comment:
Not sure, but I think we need an absolute URL here to the documentation;
https://docs.docker.com/reference/commandline/run/#setting-ulimits-in-a-container